src/rules/no-template-arrow.ts

```diff
@@ -20,8 +20,7 @@ const rule: Rule.RuleModule = {
     messages: {
       noArrow:
         'Arrow functions must not be used in templates, ' +
-        'a method should be passed directly like `${this.myMethod}` as it ' +
-        'will be bound automatically.'
+        'a method should be bound once in the constructor and passed directly like `${this.myMethod}`.'
```

Comment:
we probably still need to mention in these messages that events are bound automatically.

something like:

> LitElement event bindings are bound automatically, otherwise a method/function should be bound outside the render method (e.g. in the constructor)